src/UI/SlotArea.cpp

```diff
@@ -1114,6 +1116,7 @@ void cSlotAreaAnvil::UpdateResult(cPlayer & a_Player)
 				Output.Empty();
 				SetSlot(2, a_Player, Output);
 				m_ParentWindow.SetProperty(0, 0, a_Player);
+				m_ParentWindow.SendSlot(a_Player, this, 2);
```

Comment:
No, it turns out that cSlotAreaTemporary::SetSlot doesn't call SendSlot (the workbench, the other cSlotAreaTemporary subclass, also called SendSlot whenever it called SetSlot).

I added a call to SendSlot at the end of SetSlot.